addons/project/static/src/components/project_right_side_panel/project_right_side_panel.xml

```diff
@@ -37,7 +37,7 @@
             </ProjectRightSidePanelSection>
             <ProjectRightSidePanelSection
                 name="'profitability'"
-                show="showProfitability or state.data.show_project_profitability_helper"
```

Comment:
Here I removed ```showProfitability``` it is not present in code base. I've tried find to it and couldn't find it.